src/pybind/mgr/cli/module.py

```diff
@@ -0,0 +1,121 @@
+"""
+CLI Module - expose Ceph-Mgr Python API via CLI
+"""
+import functools
+import inspect
+import json
+import logging
+import re
+from threading import Event
+from typing import Callable, Any
+
+from mgr_module import MgrModule, HandleCommandResult, CLICommand, API
+
+
+logger = logging.getLogger()
+
+class CephCommander:
+    """
+    Utility class to inspect Python functions and generate corresponding
+    CephCommand signatures (see src/mon/MonCommand.h for details)
+    """
```

Comment:
Like forever, I wanted to have this class to automatically convert python signatures to mon commands. 

Would be great to have this also work standalone and directly usable for me.

Reply:
Probably we could take this one and came up with an alternative to `@CLICommand` that auto-guesses the signature (when Ricardo wrote `CLICommand` we couldn't use yet type annotations).